helpers.lua

```diff
@@ -808,6 +837,16 @@ function edit_user()
     end
 end
 
+-- hash the user password using sha-512 and using {CRYPT} to uses linux auth system
+-- because ldap doesn't support anything stronger than sha1
+function hash_password(password)
+    -- TODO is the password checked by regex? we don't want to
+    -- allow shell injection
+    local mkpasswd  = io.popen("mkpasswd --method=sha-512 '" ..password.."'")
```

Comment:
Well, technically there is this thing that exists but it hasn't been updated since ... 2004 and it's not packaged in debian (so it will probably never be) http://files.luaforge.net/releases/lua-crypt/lua-crypt/lua-crypt1.0

We could experiment and migrate to that later on once we managed to take back control on our debian toolchain, but that's not for today and it's more urgent to fix this.